test/evergreen.yml

```diff
@@ -5553,6 +5559,59 @@ buildvariants:
   tasks:
     - name: examples-c-tsan
 
+# This variant runs all possible tasks under TSan but instructs TSan to not report any issues it
+# encounters. This allows us to check whether a TSan change may cause other unexpected failures
+# while also not having to fix every TSan issue discovered. At the conclusion of the TSan work this
+# variant will be deleted.
+- name: ubuntu2004-tsan-no-throw
+  display_name: "~ Ubuntu 20.04 TSAN (No throw)"
+  patch_only: true
+  run_on:
+  # Defaulting to ubuntu2004-large. We can move back to ubuntu2004-test as we ID which tests can fit on the smaller host
```

Comment:
```suggestion
  # Defaulting to ubuntu2004-large. We can move back to ubuntu2004-test as we identify which tests can fit on the smaller host.
```

Reply:
I decided to remove this line instead. We added it as a placeholder while getting all the tests to pass, but a majority of the tests require the larger host to avoid timeout and OOM issues.
Since this variant is `patch_only` I think we can leave everything on the larger variant without too many concerns about cost